base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1357,19 +1369,17 @@ where
     }
 
     /// Creates a transaction to burn some Tari
```

Comment:
I think we need to explain _claim public key_ a bit, maybe something like this:
```suggestion
    /// Creates a transaction to burn some Tari. The optional _claim public key_ parameter is used in the challenge of the 
    corresponding optional _ownership proof_ return value. Burn commitments and ownership proofs will exclusively be 
    used in the 2nd layer (DAN layer). When such an _ownership proof_ is presented later on as part of some transaction 
    metadata, the _claim public key_ can be revealed to enable verification of the _ownership proof_ and the transaction 
    can be signed with the private key corresponding to the claim public key.
```

Reply:
thanks